CONTENT_STYLE_GUIDE.md

```diff
@@ -1,46 +1,321 @@
+
+### Line breaks
+
+Use a single newline to separate block-level content like headings, lists, images, code blocks, etc. The exception is second-level headings, where it should be two newlines. This helps visualize the overall structure of content in a code editor.
+
+### Headings
+
+- Use ordered headings to provide a meaningful high-level outline of your content.
+- There should be only one first-level heading per page. Blog posts don't need first level headings, as Jekyll will automatically convert the title section of your post's front matter into a first-level heading.
+- For non-blog post content, use pound/hash signs (`#`), not underlines (`---` or `===`) to designate first-level headings.
+- Use sentence case for headings (e.g. Quick Tip: Don't auto-play video, music and more).
+- Try to not use headings level 4 through 6. If your content is that detailed, it may need to be broken into separate articles.
+
+### Paragraphs
+
+- Try to keep your paragraphs on the shorter side. 5 to 6 sentences maximum.
+- Don't indent your first paragraph with space characters (e.g. `⋅⋅⋅Three spaces before a paragraph will indent it.`).
+
+### Lists
+
+- Put a period at the end of every list item.
+- Uses dashes (`-`) for unordered lists.
+- Use the number one (`1.`) for ordered lists.
+- Use one space after a list item.
+- Indent nested lists with four spaces (e.g. `⋅⋅⋅⋅-`).
+
+### Links
+
+- Links should provide context for the content they link to. Avoid using ambiguous terms like "click here".
+- Use Markdown-style links (`[link text](URL)`) instead of HTML.
+- Links should not open in new tabs or windows.
+
+### Images
+
+- Use Markdown-style images (`![alternate description](image url)`) instead of HTML.
+- Provide meaningful alternative (alt) descriptions for images. Alt descriptions should concisely describe the image's content.
+- Use complete sentences for your alt descriptions (e.g. `![A happy-looking Labrador Retriever puppy sitting in a clay flower pot.](image url)`). Including punctuation in your alt description will help some assistive technology pronounce it clearly.
+- Do not use ambiguous terms like "image", "ScreenCapture at Wed Aug 22", "post image", etc.
+- Do not use height and width attributes.
+
+### Code
+
+- Use single backticks to enclose inline code (e.g. The `footer` element typically contains metadata about its section.).
+- Use triple backticks before and after a multi-line block of code.
+- Do not use multi-line blocks of code to create diagrams, flowcharts, or other illustrations.
+
+### Horizontal rules
+
+- Use three hyphens (`---`) to create a horizontal rule.
+- Use horizontal rules to denote thematic breaks in content
+- Do not use horizontal rules for decoration.
+
+### Inline HTML
+
+Use HTML only when Markdown cannot accurately describe your content. Use relevant, semantic HTML elements and attributes. Examples of this would be a video embed or a definition list.
```

Comment:
mention `<kbd>` and that other text-level semantics may not have markdown equivalents